apps/estuary/test/unit/data_reader_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Estuary.DataReaderTest do
+  use ExUnit.Case
+
+  import Mox
+
+  alias Estuary.DataReader
+
+  setup :set_mox_global
+  setup :verify_on_exit!
+
+  test "should read the event" do
```

Comment:
Is this name correct?

Reply:
^ We're going to go ahead and delete this guy.